packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -376,6 +379,7 @@ export class BlobManager {
 			// eventually and wait. We do this even if the local client doesn't have the blob payloadPending flag
 			// enabled, in case a remote client does have it enabled. This wait may be infinite if the uploading
 			// client failed the upload and doesn't exist anymore.
+			// eslint-disable-next-line require-atomic-updates
```

Comment:
Nit: would be good to add a quick explanation for disabling the rule (or a TODO if we want to come back to this later).

Reply:
TBH these were "prefer inline disables over package-wide disables to get coverage for future changes", I didn't examine closely.  I'm making changes to BlobManager in particular soon that should clean these up though.

My mentality is that any eslint disable is de facto a TODO already, lmk if you'd still prefer explicit TODO comments though

Reply:
I think explicit TODOs are good. I suspect most people see disables and assume them to be intentional / desired. The TODO better signals that it isn't and should be fixed.